examples/model_simulated/vrops_bike_model_simulated.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's use this refactoring opportunity to remove all references to vRops. Rename it to Vmware_Ice